mono/mini/interp/transform.c

```diff
@@ -7114,7 +7128,7 @@ interp_cprop (TransformData *td)
 				} else {
 					locals [dest_local].type = STACK_VALUE_NONE;
 				}
-			} else if (sp->val.type == STACK_VALUE_NONE) {
+			} else if (sp->val.type == STACK_VALUE_NONE || sp->val.type == STACK_VALUE_ARG) {
```

Comment:
not sure about this. Is that okay, for now at least? (until we do more stuff with `STACK_VALUE_ARG`).

Reply:
doing copy propagation through arguments requires `starg` to be a reasonably common opcode, which I really doubt.